src/sentry/api/bases/organization_events.py

```diff
@@ -436,25 +434,25 @@ def handle_error_upsampling(self, project_ids: Sequence[int], results: dict[str,
             fields_meta = results.get("meta", {}).get("fields", {})
 
             for result in data:
```

Comment:
Please add some doc here.

Reply:
If you can make it a simple function that iterates over some known keys that need this.